### ISSUE TITLE
Public container crashes

### ISSUE THREAD

**Issue by JohnRusk** [little-angry-clouds/prometheus-data-generator#2]
The Docker image at `littleangryclouds/prometheus-data-generator` appears to include an old version of the code.  It gives this error when I run it:

```
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "/opt/prometheus_data_generator/main.py", line 130, in update_metrics
    timeout = time.time() + sequence["eval_time"]
KeyError: 'eval_time'
```

The line its complaining about does not exist in the current master branch.  When I rebuild my own container image from source, it all works fine. So I think all that is necessary is for the `littleangryclouds/prometheus-data-generator` to be updated with a fresh build from the latest source. ~(But note, there might need to be an unrelated bug fix to the source before the container is rebuilt. I'll post a second issue shortly).~

**Comment by JohnRusk** [comment]
The followup issue is #3

**Comment by alexppg** [comment]
`eval_time` is a required parameter. It is reflected in the `README` in the [supported keywords](https://github.com/little-angry-clouds/prometheus-data-generator#supported-keywords), but not in the examples and some configurations. I've changed in [this commit](https://github.com/little-angry-clouds/prometheus-data-generator/commit/48c948ab375e4153565ae52f6816688ee6bcaff1). 
Is it clearer now? @JohnRusk

**Comment by JohnRusk** [comment]
Yes, that does clarify it, thanks.

BTW, for some reason, my local branch wasn't up to date. I.e. it did not contain the commits where you changed the variable name in the code.  That's why I thought the readme was correct and the public image was wrong.  I'm not sure how I ended up with my forked repo being behind yours.  Now that I've got mine up to date, I can see that I was wrong about the original formulation of this issue, and that your documentation fix is indeed all that was required.  Thanks :-)
